### PR TITLE
Document the line-height feature added in 5.5 by TINY-4843

### PR DESCRIPTION
**Pull request by ashfordneil**
Related Ticket: TINY-4843

Description of Changes:
* Document the line height feature coming with tiny 5.5

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: advanced/editor-command-identifiers.md [hunk 1]
```diff
@@ -500,16 +502,19 @@ The following command values can be queried using the [queryCommandValue]({{ sit
 
 | Command       | Description                                            |
 | --------------| ------------------------------------------------------ |
-| FontName      | Returns the font name of the current selection.        |​
-| FontSize      | Returns the font size of the current selection.        |​
+| FontName      | Returns the font name of the current selection.        |
+| FontSize      | Returns the font size of the current selection.        |
 | ToggleSidebar | Returns the current state of sidebar (open or closed). |
+| LineHeight    | Returns the line height of the current selection.      |
```
Comment (lnewson) [inline advanced/editor-command-identifiers.md:508]: Maybe put this above, so it's in alphabetical order?

File: release-notes/release-notes55.md [hunk 2]
```diff
@@ -41,6 +41,12 @@ The new listbox component for custom dialogs is a composite component with a lab
 
 For information on the listbox dialog component, see: [Dialog components - listbox]({{site.baseurl}}/ui-components/dialogcomponents/#listbox).
 
+### New line height support in the editor
+
+{{site.productname}} 5.5 now supports line height. There is a new `lineheight` button available for toolbars, and a new `lineheight` item available for menus (included by default in the format menu). As well as this, there is a new `LineHeight` editor command to change line height and a `LineHeight` query command value to get the current line height.
```
Comment (lnewson) [inline release-notes/release-notes55.md:46]: `menu item`?

**Review by lnewson — APPROVED** [review]

**Review by EkimChau — APPROVED** [review]

File: advanced/editor-command-identifiers.md (hunk 1 above)
Comment (ashfordneil) [reply advanced/editor-command-identifiers.md:508]: will do. I also just realised that this doesn't have a "5.5 only" warning

**Review by HAFRMO — APPROVED** [review]

File: _includes/configuration/lineheight-formats.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+## lineheight_formats
+
+{{ site.requires_5_5v }}
+
+This option allows you to override the line heights displayed in the `lineheight` dropdown toolbar and the `lineheight` menu item. Each item in the string should be space separated.
```
Comment (tylerkelly13) [inline _includes/configuration/lineheight-formats.md:5]: ... on the `lineheight` dropdown toolbar button and ....

File: release-notes/release-notes55.md [hunk 4]
```diff
@@ -41,6 +41,12 @@ The new listbox component for custom dialogs is a composite component with a lab
 
 For information on the listbox dialog component, see: [Dialog components - listbox]({{site.baseurl}}/ui-components/dialogcomponents/#listbox).
 
+### New line height support in the editor
+
+{{site.productname}} 5.5 now supports line height. There is a new `lineheight` button available for toolbars, and a new `lineheight` menu item (included by default in the format menu). As well as this, there is a new `LineHeight` editor command to change line height and a `LineHeight` query command value to get the current line height.
```
Comment (tylerkelly13) [inline release-notes/release-notes55.md:46]: now supports adjustable line heights.

Comment (tylerkelly13) [inline release-notes/release-notes55.md:46]: s/As well as this, there is/There is also/

File: release-notes/release-notes55.md [hunk 5]
```diff
@@ -41,6 +41,12 @@ The new listbox component for custom dialogs is a composite component with a lab
...
+
+For information on the toolbar button, see: [Available Toolbar Buttons - The core toolbar buttons]({{site.baseurl}}/advanced/available-toolbar-buttons/#thecoretoolbarbuttons). For information on the new menu item, see: [Available Menu Items - The core menu items]({{site.baseurl}}/advanced/available-menu-items/#the-core-menu-items). For information on the new editor command, see: [Commands available for {{site.productname}} - Core Editor commands]({{site.baseurl}}/advanced/editor-command-identifiers/#coreeditorcommands). For information on the new query command value, see: [Commands available for {{site.productname}} - Available query command values]({{site.baseurl}}/advanced/editor-command-identifiers/#availablequerycommandvalues).
```
Comment (tylerkelly13) [inline release-notes/release-notes55.md:48]:
```suggestion
For information on:

- The `lineheight` toolbar button, see: [Available Toolbar Buttons - The core toolbar buttons]({{site.baseurl}}/advanced/available-toolbar-buttons/#thecoretoolbarbuttons).
- The`lineheight` menu item, see: [Available Menu Items - The core menu items]({{site.baseurl}}/advanced/available-menu-items/#the-core-menu-items).
- The `LineHeight` editor command, see: [Commands available for {{site.productname}} - Core Editor commands]({{site.baseurl}}/advanced/editor-command-identifiers/#coreeditorcommands).
- The `LineHeight` query command value, see: [Commands available for {{site.productname}} - Available query command values]({{site.baseurl}}/advanced/editor-command-identifiers/#availablequerycommandvalues).
```

**Review by tylerkelly13 — CHANGES_REQUESTED** [review]
Just some minor changes needed, otherwise looks good!

**Review by tylerkelly13 — APPROVED** [review]

**Pull request merged**